asset/CMakeLists.txt

```diff
@@ -7,9 +7,9 @@ if (MSVC)
 	set(ICON "${CMAKE_CURRENT_SOURCE_DIR}/OpenBoardView.rc" PARENT_SCOPE)
 elseif(APPLE)
 	# Used for app bundle
-	set(ICON "${CMAKE_CURRENT_SOURCE_DIR}/brd.icns" PARENT_SCOPE)
-	set_source_files_properties("brd.icns" PROPERTIES
-		MAXOSX_PACKAGE_LOCATION "Resources")
```

Comment:
This is broken because is uses `MAXOSX` instead of `MACOSX` - not because it's in the wrong place